metrics/tenant_registries.go

```diff
@@ -443,53 +461,37 @@ func sum(mf *dto.MetricFamily, fn func(*dto.Metric) float64) float64 {
 	return result
 }
 
-// max returns the maximum value from all metrics from same metric family (= series with the same metric name, but different labels)
-// Supplied function extracts value.
-func max(mf *dto.MetricFamily, fn func(*dto.Metric) float64) float64 {
-	return fold(mf, fn, func(val, res float64) float64 {
-		if val > res {
-			return val
-		}
-		return res
-	})
-}
-
-// min returns the minimum value from all metrics from same metric family (= series with the same metric name, but different labels)
-// Supplied function extracts value.
-func min(mf *dto.MetricFamily, fn func(*dto.Metric) float64) float64 {
-	return fold(mf, fn, func(val, res float64) float64 {
-		if val < res {
-			return val
-		}
-		return res
-	})
-}
-
 // fold returns value computed from multiple metrics, using folding function. if there are no metrics, it returns 0.
```

Comment:
The part of the comment which says "if there are no metrics, it returns 0" doesn't look correct anymore. Now it returns NaN.

Reply:
Done in https://github.com/grafana/dskit/pull/368/commits/661d25d7e6718d7ce9fa62e69c82b6b8c6957cb7.